app/models/rubygem.rb

```diff
@@ -33,9 +33,12 @@ class Rubygem < ApplicationRecord
   after_create :create_gem_download
   before_destroy :mark_unresolved
 
+  # TODO: Change MFA_RECOMMENDED_THRESHOLD to 150_000_000 and MFA_REQUIRED_THRESHOLD to 165_000_000
   MFA_RECOMMENDED_THRESHOLD = 165_000_000
+  MFA_REQUIRED_THRESHOLD = 180_000_000
```

Comment:
Double checking here: @jenshenny, you're implying `MFA_REQUIRED_THRESHOLD` is arbitrary for now because we're [in the midst of realigning](https://github.com/rubygems/rubygems.org/pull/3089#discussion_r898305452) on whether the thresholds are 150M/165M vs. 165M/180M. Is that correct?

Reply:
Mostly yeah. Since we are deciding between 165/180M downloads for required, 180 was chosen for now because `MFA_REQUIRED_THRESHOLD` can't be set to 165M because `MFA_RECOMMENDED_THRESHOLD` has to be strictly less than the `MFA_REQUIRED_THRESHOLD` (tests will break, etc).